### PR TITLE
feat: Created homepage of Quizie

### PR DESCRIPTION
**Pull request by arjunbharti**
Homepage of Quizie:

- Quick intro
- View different quiz categories

Screenshot from 2022-02-17 17-40-44

Screenshot from 2022-02-17 17-41-11

### CODE REVIEW

File: styles/homepage.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.header{
+    padding-top: 2rem;
+    font-weight: 600;
+}
+
+.main-content{
```
Comment (tushargoyal812) [inline styles/homepage.css:6]: suggestion add a space before { use beautify extension in vs code

**Review by tushargoyal812 — COMMENTED** [review]
good design,only minor changes needed and looks good to merge

**Review by poojakelkar — COMMENTED** [review]
Looks Good to merge for me

File: styles/homepage.css (hunk 1 above)
Comment (arjunbharti) [reply styles/homepage.css:6]: thanks, will take care of it

**Pull request merged**